gravitee-apim-console-webui/src/portal/navigation/portal-navigation.service.ts

```diff
@@ -60,6 +60,13 @@ export class PortalNavigationService {
         icon: 'gio:report-columns',
       });
     }
+    if (this.permissionService.hasAnyMatching(['environment-api-r', 'environment-api-u'])) {
```

Comment:
Should be settings permissions, not apis

Reply:
Done